var/spack/repos/builtin/packages/rocm-debug-agent/package.py

```diff
@@ -37,19 +38,21 @@ def url_for_version(self, version):
     variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
 
     depends_on('cmake@3:', type='build')
-    depends_on("elfutils", type='link')
+    depends_on('elfutils@:0.168', type='link')
```

Comment:
Any reason for this?

Reply:
rocm-debug-agent Build doesn't support newer versions of elfutils right now. The Build errors out.

Reply:
ok, can you report that somewhere upstream?